runtime/v2/module.go

```diff
@@ -275,6 +275,7 @@ func ProvideEnvironment(logger log.Logger, config *runtimev2.Module, key depinje
 		EventService:    stf.NewEventService(),
 		GasService:      stf.NewGasMeterService(),
 		HeaderService:   stf.HeaderService{},
+		RouterService:   stf.NewRouterService(kvService, app.app.queryRouterBuilder, app.app.msgRouterBuilder),
```

Comment:
Would it be cleaner to just place `MessageRouterService` and `QueryRouterService` as fields on Environment instead of nesting them behind a getter in `RouterService`?

Reply:
Something we should do consistently on main afterwards given we have the same on main.

Reply:
makes senes to me i shall flag a TODO